recipe/gdal_version_win.patch

```diff
@@ -0,0 +1,11 @@
+--- setup.py	2016-06-15 13:28:44.000000000 -0300
++++ setup.py	2016-06-15 20:03:19.706422409 -0300
+@@ -79,6 +79,8 @@
+     index = sys.argv.index('--gdalversion')
+     sys.argv.pop(index)
+     gdalversion = sys.argv.pop(index)
++else:
++   gdalversion = '2.1.0'
```

Comment:
@sgillies I hacked my way here on  Windows b/c `--gdalversion` did not work. Maybe we could use:

```python
from osgeo import gdal
gdalversion = gdal__version__
```

I can sed a PR if that is acceptable.